cassandra2/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2012 - 2015 YCSB contributors. All rights reserved. Licensed under 
```

Comment:
looks like missing line breaks between copyright and license as well as within license.